README.md

```diff
@@ -32,11 +32,9 @@ Utilizing the SDK requires an RPC provider:
 ```js
 import BadgerSDK, { Network } from '@badger-dao/sdk';
 
-const provider = new ethers.providers.JsonRpcProvider('https://myrpc.io/');
-
 const sdk = new BadgerSDK({
   network: Network.Ethereum,
-  provider
+  provider: 'https://myrpc.io/',
```

Comment:
Is this just a placeholder?  Don't see anything about `myrpc.io` anywhere on the interwebs

Reply:
we can change it to a real one if you want like cloudflare or something

Reply:
ok i'll update that in a later fix since this one has the approval rn